fmriprep/workflows/bold/t2s.py

```diff
@@ -76,6 +76,15 @@ def init_bold_t2s_wf(bold_echos, echo_times, mem_gb, omp_nthreads,
             the skull-stripped optimal combination mask
     """
     workflow = Workflow(name=name)
+    workflow.__desc__ = """\
+Voxel-wise T2* was estimated from preprocessed BOLD by fitting to a
+monoexponential signal decay model with log-linear regression. For each voxel,
```

Comment:
In markdown and restructured text we usually try to start each sentence with a new line. That way it is easier to see diffs if we change this in a future.